demo/src/demos/PageLayout/PageLayoutStickyBarDemo.tsx

```diff
@@ -0,0 +1,100 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Add test for this file to get screenshots.

Reply:
I have added test